app/services/accredited_provider/importer.rb

```diff
@@ -0,0 +1,53 @@
+class AccreditedProvider::Importer
+  include ServicePattern
+
+  IMPORTABLE_KEYS = %w[
+    code
+    name
+    provider_type
+    ukprn
+    urn
+    email
+    telephone
+    website
+    street_address_1
+    street_address_2
+    street_address_3
+    city
+    county
+    postcode
+  ].freeze
+
+  def initialize(updated_since: nil)
+    @updated_since = updated_since
+  end
+
+  attr_reader :updated_since
+
+  def call
+    errors = []
+    accredited_providers = ::AccreditedProvider::Api.call(updated_since:)
+    accredited_providers.each do |provider_details|
+      provider = generate_provider(provider_details)
+      unless provider.save
```

Comment:
This is just a question, not a request for change necessarily. Bit with the School importer, we are collecting the data and then upserting for performance. Are we concerned about the performance implications of saving all the records individually, or we do think it is too small of a data set to worry about?

Reply:
resolved in [Refactor importer to align with GiasImporter](https://github.com/DFE-Digital/itt-mentor-services/pull/92/commits/95894fbc58176adec76570e9578c60a49761f274)